ngi_pipeline/utils/filesystem.py

```diff
@@ -100,11 +100,21 @@ def execute_command_line(cl, shell=False, stdout=None, stderr=None, cwd=None):
     return p_handle
 
 def do_symlink(src_files, dst_dir):
+    do_link(src_files, dst_dir, 'soft')
+
+def do_hardlink(src_files, dst_dir):
```

Comment:
sounds like me this kind of things.... I like have meaningful function names that allow to "read the code" instead of look to options (i.e. link_type).